packages/front-end/components/Admin/EditOrganization.tsx

```diff
@@ -96,6 +177,63 @@ const EditOrganization: FC<{
             onChange={(e) => setOwner(e.target.value)}
           />
         </div>
+        <div className="mt-3">
+          Verified Domain
+          <input
+            type="text"
+            className="form-control"
+            value={verifiedDomain}
+            onChange={(e) => setVerifiedDomain(e.target.value)}
+          />
+          <span className="text-muted small">
+            This is used so new users can auto join this org by matching email
+            domain.
+          </span>
+        </div>
+        <div className="mt-3">
+          Auto approve members with email domain
+          <Toggle
+            className="ml-2"
+            id="autoApproveMembers"
+            value={autoApproveMembers}
+            setValue={setAutoApproveMembers}
+          />
+        </div>
+        {isCloud() ? (
+          <>
+            <div className="mt-3">
+              Free Seats
+              <input
+                type="number"
+                min={0}
+                className="form-control"
+                value={freeSeats}
+                onChange={(e) => setFreeSeats(parseInt(e.target.value))}
+              />
+              <div>
+                <span className="text-muted small">
+                  Number of seats that will not be billed
```

Comment:
`Number of seats that can be added when on a free plan`

Reply:
done